cernopendata/modules/fixtures/data/docs/cms-releases-heavy-ion-data/cms-releases-heavy-ion-data.md

```diff
@@ -0,0 +1,32 @@
+The CMS experiment at CERN has released into the public domain its first batches of open data from heavy-ion collisions at the Large Hadron Collider.
+The two batches contain two datasets from 2010 and four from 2011, recorded during the LHC’s first data-taking period (“Run 1”), along with eight sets of reference data from proton–proton collisions recorded at the same collision energy.
+The total disk volume of the release is 214&nbsp;TB.
+
+Famous for the proton–proton collisions that resulted in the discovery of the Higgs boson, the LHC also collides nuclei from heavy elements (heavy ions) for a few weeks each year, complementing data from proton–proton interactions.
+Heavy-ion collisions at the LHC occur at the highest energy achieved in the laboratory.
+They can recreate the very early conditions of the universe, when the quarks and gluons, which are normally bound together to form particles like protons and neutrons, existed freely in a soup-like state known as a quark–gluon plasma.
+Scientists at the LHC can use such collisions to understand these early-universe conditions and study the interaction of other particles with the quark–gluon plasma.
+
+In 2010 and 2011, the heavy-ion collisions were between nuclei of lead.
+Using these data, CMS could observe phenomena several signatures of the quark–gluon plasma, including the imbalance between the momenta of each jet of particles produced in a pair, the suppression (“quenching”) of particle jets in jet–photon pairs and the “melting” of certain composite particles.
+
+
+CMS has already released 100% of the proton–proton data the collaboration recorded in 2010 and 2011.
+Therefore, with the release of the lead–lead data, all of the data CMS recorded in the first two years of LHC operation is in the public domain, in accordance with the collaboration open-data policy.
+
+Analysing data from the LHC requires specialised software and analysis environments.
+CMS is providing these in the form of Virtual Machine images as well as Docker container images, which come with all the necessary software.
+Further, in order for a new data explorer to familiarise themself with the kinds of analysis one can perform with the data, CMS has also provided usage examples.
+Those interested in specific advice can ask for it on the [official CERN Open Data forum](https://opendata-forum.cern.ch).
+
+“Preparing this release presented unique challenges to CMS,” notes Kati Lassila-Perini, who has lead the CMS Data Preservation and Open Access team since 2012.
```

Comment:
lead -> led ?

Reply:
Yes, should be "led". Data are plural. :) Will update the PR.

Reply:
Done.